tests/src/3d/testqgs3drendering.cpp

```diff
@@ -398,6 +399,59 @@ void TestQgs3DRendering::testExtrudedPolygons()
   QVERIFY( renderCheck( "polygon3d_extrusion", img, 40 ) );
 }
 
+void TestQgs3DRendering::testExtrudedPolygonsDataDefined()
+{
+  QgsPropertyCollection propertyColection;
+  QgsProperty diffuseColor;
+  QgsProperty ambientColor;
+  QgsProperty specularColor;
+  diffuseColor.setExpressionString( QStringLiteral( "'blue'" ) );
```

Comment:
This test would be more useful if we included a conditional statement in here, so that we're certain that not all features are getting the same result.

Reply:
![polygon3d_extrusion_data_defined](https://user-images.githubusercontent.com/7416892/97827459-bf013200-1c9a-11eb-9271-fda5236cb43a.png)

Reply:
Thanks!